src/models/discreteBar.js

```diff
@@ -216,6 +217,7 @@ nv.models.discreteBar = function() {
 
     chart._options = Object.create({}, {
         // simple options, just get/set the necessary values
+        barWidth:{get: function(){return barWidth;}, set: function(_){barWidth=_;}},
```

Comment:
barWidth only makes sense to be between 0 and 1 here right?  So it should probably validate that here.

Reply:
@liquidpele if it doesn't validate, should I just return the default value or do a console.error?

Reply:
I would nv.log it (wrapper around console.log we use) and ignore the passed value.